pyomo/core/expr/taylor_series/taylor.py

```diff
@@ -0,0 +1,22 @@
+from pyomo.core.expr.current import identify_variables, value
```

Comment:
I would advocate moving `taylor.py` to `pyomo/core/expr/taylor_series.py` and git rid f the extra directory.